core/tests/performance_tests/lda_string_classifier_performance_test.py

```diff
@@ -130,7 +131,8 @@ def predict(self, num):
             raise Exception('No classifier found')
         classifier = (
             classifier_registry.Registry.get_classifier_by_algorithm_id(
-                feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput']))
+                feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput'][
```

Comment:
this should be generic in the sense that the interaction type should not be hard-coded. But it is OK to fix this in another PR perhaps. Just take a note of it for now.